vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -767,6 +767,37 @@ export class DuplicateInstallDetected extends DotnetCustomMessageEvent {
     public readonly eventName = 'DuplicateInstallDetected';
 }
 
+export abstract class WebRequestTimer extends DotnetCustomMessageEvent {
+
+    constructor(public readonly eventMessage: string, public readonly durationMs: string,
+        public readonly success: string, public readonly url: string
+    ) { super(eventMessage); }
```

Comment:
Why do you not need to set anything from the other parameters?

Reply:
Good question! Those will get auto-populated as members of the class as required by the constructor. The only other one goes into the parent / super constructor the same way.